docs/settings.rst

```diff
@@ -5,6 +5,14 @@ Settings
 
 pulp_rpm adds configuration options to the those offered by pulpcore.
 
+KEEP_CHANGELOG_LIMIT
```

Comment:
Does it make sense to add that we keep **the most recent** changelogs? 
It's likely obvious, so I'm not sure if it's worth adding it. Just sharing a thought.

Reply:
I agree it can be made more clear